torchtitan/parallelisms/pipelining_utils.py

```diff
@@ -45,10 +45,25 @@ def build_pipeline_schedule(job_config, parallel_dims, stages, loss_fn):
     if n_microbatches is None:
         n_microbatches = job_config.experimental.pipeline_parallel_degree
 
+    if job_config.experimental.pipeline_parallel_schedule == "zb":
+        stage_index_to_group_rank = {
```

Comment:
nit: there should be a helper fn (in this file iirc) that produces this mapping for you.  maybe its not in the right format though?  i also wonder if we should land that helper to pytorch in some way.